src/lib/snack-bar/snack-bar-container.ts

```diff
@@ -11,6 +18,20 @@ import {
 } from '../core';
 import {MdSnackBarConfig} from './snack-bar-config';
 import {MdSnackBarContentAlreadyAttached} from './snack-bar-errors';
+import {Observable} from 'rxjs/Observable';
+import {Subject} from 'rxjs/Subject';
+
+
+export class SnackBarState {
```

Comment:
Sure thing, I was thinking that this way allowed us to have a place to hold the values centrally, rather than strings in different places.